pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -92,7 +92,7 @@ func (db *DB) init() (err error) {
 		return err
 	}
 
-	_, err = tx.Exec("CREATE TABLE IF NOT EXISTS `bandwidth_agreements` (`satellite` BLOB, `agreement` BLOB, `signature` BLOB);")
+	_, err = tx.Exec("CREATE TABLE IF NOT EXISTS `bandwidth_agreements` (`satellite` BLOB, `uplink` BLOB, `serialnum` TEXT, `total` INT(10), `maxsize` INT(10), `createdunixsec` INT(10), `expirationunixsec` INT(10), `action` TEXT, `daystartdateunixsec` INT(10), `agreement` BLOB, `signature` BLOB);")
```

Comment:
'daystartdateunixsec' etc are hard to read. Could we use underscores to separate words?  Googling indicates Postgres's identifier limit is 63 characters and Sqlite doesn't have a hard limit.